src/utils/helpers.js

```diff
@@ -0,0 +1,9 @@
+export const filteredMovieData = movies => {
+  return movies.map(movie => ({
```

Comment:
This is good info, and once we get everything set up, we can see if we want more. Thank you for this!